### PR TITLE
Fix acessibilidade

### PR DESCRIPTION
**Pull request by idgserpro**

### CODE REVIEW

File: src/brasil/gov/tiles/tiles/nitf.py [hunk 1]
```diff
@@ -96,11 +96,7 @@ def alt(self):
 
         https://github.com/plonegovbr/brasil.gov.tiles/blob/caa250e9c3428d000174368321c42dae2fb48f92/src/brasil/gov/tiles/tiles/basic.py#L160
         """
-        return any((
-            self.data.get('image_description'),
-            self.data.get('description'),
-            self.data.get('title'),
-        ))
+        return self.data.get('image_description') or self.data.get('description') or self.data.get('title')
```
Comment (hvelarde) [inline src/brasil/gov/tiles/tiles/nitf.py]: why?

File: src/brasil/gov/tiles/tiles/templates/nitf.pt [hunk 2]
```diff
@@ -25,13 +25,13 @@
       <tal:image
           condition="python:field['id'] == 'image'"
           define="scale python:field.get('scale', 'large');
-                  position python:field.get('position', '');">
-        <a class="imag" tal:attributes="href view/getURL;
+                  position python:field.get('position', '');
+                  scales view/@@images;
```
Comment (hvelarde) [inline src/brasil/gov/tiles/tiles/templates/nitf.pt]: I think the `@@images` view must be called from the `context` and not from the `view`.

File: src/brasil/gov/tiles/tests/test_social.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+from zope.interface.verify import verifyObject
+
+
+class SocialTileTestCase(BaseIntegrationTestCase):
+
+    def setUp(self):
+        super(SocialTileTestCase, self).setUp()
+        self.tile = self.portal.restrictedTraverse(
+            '@@{0}/{1}'.format('social', 'test-tile')
+        )
```
Comment (hvelarde) [inline src/brasil/gov/tiles/tests/test_social.py]: move this to the line above.

File: src/brasil/gov/tiles/upgrades/v4003/configure.zcml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    xmlns:gs="http://namespaces.zope.org/genericsetup"
+    i18n_domain="brasil.gov.tiles">
+
+  <gs:upgradeSteps
+      source="4002"
+      destination="4003"
+      profile="brasil.gov.tiles:default">
+
+    <gs:upgradeStep
+        title="Cook css"
```
Comment (hvelarde) [inline src/brasil/gov/tiles/upgrades/v4003/configure.zcml]: `CSS` all over the place.

File: CHANGES.rst [hunk 5]
```diff
@@ -4,7 +4,15 @@ Alterações
 1.4b2 (unreleased)
 ^^^^^^^^^^^^^^^^^^
 
-- Nothing changed yet.
+- Altera o tile de redes sociais para retirar o link das abas Facebook e Twitter para adequar a
```
Comment (hvelarde) [inline CHANGES.rst]: documentation is not code, make this a one-lines.

File: src/brasil/gov/tiles/tiles/nitf.py (hunk 1 above)
Comment (idgserpro) [reply src/brasil/gov/tiles/tiles/nitf.py]: This was wrong: `any` in Python [returns a boolean](https://docs.python.org/2/library/functions.html#any) and not the value of what's inside the parenthesis if anyone of them it's not `False`.

File: src/brasil/gov/tiles/tiles/templates/nitf.pt (hunk 2 above)
Comment (idgserpro) [reply src/brasil/gov/tiles/tiles/templates/nitf.pt]: We're using the same logic applied to https://github.com/collective/collective.nitf/blob/7160c02a743c4c162a91e5310934b28f26b49b53/src/collective/nitf/tiles/nitf.pt#L34. Should be really changed?

Comment (hvelarde) [reply src/brasil/gov/tiles/tiles/templates/nitf.pt]: I have no idea.

**Pull request merged**